### ISSUE TITLE
MSI build failing

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#4890]
The MSI build of the current Debrief release is failing:

```
Checking the event log we have "Error 1335. 
The cabinet file 'Debrief.cab' required for this installation is corrupt and cannot be used"
```

Would you mind repeating the investigation/fix you did a few weeks ago?

**Comment by saulhidalgoaular** [comment]
Good afternoon @IanMayo 

I would mask this ticket as non-reproducible. This is what I have found in the research:

*  Master in last release tag create a correct msi locally, but somehow travis generates a corrupted one.
* Develop creates a corrupted msi
* I did a bisect on master, and It passes from bad to good ın the following commits.
  Bad: a0e15ac5
  Good: 1fa3861c

* I did bisect on develop, and it passes from bad to good in the followings commits.
  Bad: 6ac562db
  Good: f4c13ebd

* I also compared the fıles from the installer fails with the files packaged inside the msi, and the following folder was missing in the fail installation
/p2/org.eclipse.equinox.p2.core/cache/binary
* I opened a corrupted msi, with lessmsi, and after I unpacked the files, I can use Debrief from those files. (This proof that the files inside the msi are all files needed)

Regards.
Saul
